Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Set the base image to Ubuntu
+FROM mysql
```

Comment:
Not sure why we need mysql. But it's fine that we start from this.

Reply:
MySQL is used in this application to store the status of the command for every user, Approved or Not Approved.